dags/infra/variables.py

```diff
@@ -8,6 +8,19 @@
     "index_landsat_scenes_sqs_queue", "deafrica-dev-eks-index-landsat-scene"
 )
 
+AFRICA_AWS_REGION = Variable.get("cape-town-region", "af-south-1")
```

Comment:
use this shared one `AWS_DEFAULT_REGION = Variable.get("aws_default_region", "af-south-1")`